src/main/java/carracinggame/domain/Car.java

```diff
@@ -0,0 +1,50 @@
+package carracinggame.domain;
+
+public class Car {
+
+    public static final int INITIAL_POSITION = 0;
+    public static final int REFERENCE_NUMBER = 4;
+
+    private String name;
+    private int position = INITIAL_POSITION;
+
+    Car(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getPosition() {
+        return this.position;
+    }
+
+    public void move(int number) {
+        if (isOverProbability(number)) {
+            position++;
+        }
+    }
+
+    private boolean isOverProbability(int number) {
+        if (number >= REFERENCE_NUMBER) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    private boolean isOverProbability(int number) {
        return number >= REFERENCE_NUMBER;
    }
```

이런 경우는 이렇게 단축해서 표현할 수 있죠!

Reply:
네! 비슷한 것들도 다 바꿨습니다.